doc/Makefile

```diff
@@ -3,10 +3,87 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
+########## Config
+
+ifndef GITDIR
+GITDIR    = ../../../projects/gtda-docs
```

Comment:
This line seems to assume a more specific folder structure than explained in https://giotto-ai.github.io/gtda-docs/dev/contributing/#basic-instructions. That is, not only should `gtda-docs` and `giotto-tda` be in the same folder, that folder should be called "projects"?

Reply:
This reflects my local directory structure. Ideally, this should be in a config file for the Makefile. Or, not defined at all, and the user would need to set the environment variable explicitly. Same for RUNJUPYTERNOTEBOOKS.

Reply:
Why not just
```
../../gtda-docs
```
?

Reply:
It can be a default value, sure